packages/api/src/routes/v3/report.ts

```diff
@@ -59,11 +63,13 @@ export default (server: FastifyInstance, _options: RouteShorthandOptions, done:
             `- <b>URL:</b> ${request.body.url}`,
             `- <b>User-Agent:</b> ${request.body.userAgent ?? '-'}`,
             `- <b>Version:</b> ${request.body.version}`,
+            `- <b>Explanation:</b> ${request.body.explanation ? request.body.explanation : '-'}`,
           ].join('\n'),
           labels: ['bug'],
           owner: environment.github.owner,
           repo: environment.github.repo,
           title: url,
+          explanation: `${request.body.explanation}`,
```

Comment:
This is not correct, we don't need to pass the explanation to the `octokit.request`. It's just a coincidence that we have this same property in our `request.body` and it's used for different purposes by `octokit`.

Reply:
Corrected.